libgloss/arc/readme-hostlink.md

```diff
@@ -0,0 +1,50 @@
+Metaware hostlink IO
```

Comment:
Moved to the top after: https://github.com/foss-for-synopsys-dwc-arc-processors/newlib/pull/14#discussion_r563747947

I think here we have a better chance that another reader will find this readme.

Reply:
Okay then. Please update this sentence:

```
This directory contains target-side implementation of Metaware hostlink
```

Because it is not correct. That's what triggered me to ask you to move it to `hl/` in the first place.

Reply:
not sure if newlib maintainers will accept an README.md. For github is nice, but not for upstream newlib. The best is to move it to ABI repo.

Reply:
Updated the doc in our repo. I think if newlib maintainers won't like it we always can move it to the ABI repo which is also good (best?) place for this doc.